src/components/FormInput/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { type } from "os";
+import { useState } from "react";
+import styled from "styled-components";
+
+interface InputBaseProps {
+  type: "password" | "text" | "email";
+}
+
+function FormInput({ type }: InputBaseProps) {
+  return <StyledFormInput type={"password"} />;
```

Comment:
```suggestion
function FormInput({ type }: InputBaseProps) {
  return <StyledFormInput type={type} />;
```

으로 되어야 할 것 같아요 지금 이상태면 password로 되어있는 거니까 !

Reply:
아 넵 ! 확인해보느라 설정해놓고 안바꿨네요, 수정하고 다시 올리겠습니다